src/site/stages/build/plugins/read-assets-from-disk.js

```diff
@@ -15,7 +15,7 @@ function readAssetsFromDisk(buildOptions) {
         filename =>
           // eslint-disable-next-line no-param-reassign
           (files[`generated/${filename}`] = {
-            path: `generated/{$filename}`,
+            path: `generated/${filename}`,
```

Comment:
I wonder if this little typo has been messing with things :thinking:

Reply:
:man_facepalming: Weird how this wasn't caught by eslint. I thought we had a rule that said "don't use template strings unless you've got a variable in there" or something.

At any rate, it may have, but the only thing I think it might have messed with would be the broken link checker. And even then, I think we're reporting those out to the #cms-team channel, so I think we would have heard about it. :grimacing: 

Odd. I'm not sure what to do with this.